### PR TITLE
Adding Bicep template for App Configuration with KV Copy

### PR DESCRIPTION
**Pull request by willvelida**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@glloyd2010f - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@glloyd2010f - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by willvelida** [comment]
@bmoore-msft Please review :)

**Comment by glloyd2010f** [comment]
@willvelida how can I get ahold of you so that I the updates can be added prior to this being merged?

**Comment by willvelida** [comment]
@glloyd2010f Do you have any email address? You can also ping me on Teams if you like?

File: quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    value: item.value
+    contentType: (contains(item, 'contentType') ? item.contentType : json('null'))
+    tags: (contains(item, 'tag') ? item.tag : json('null'))
+  }
+  dependsOn: [
+    configStore
+  ]
+}]
+
+output KeyVauleReference string = reference(resourceId('Microsoft.AppConfiguration/configurationStores/keyValues', configStoreName, keyData[0].key), '2021-10-01-preview').value
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep:60]: use the bicep syntax for the property references

File: quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    name: 'standard'
+  }
+}
+
+resource configStoreNameKeyVaule 'Microsoft.AppConfiguration/configurationStores/keyValues@2021-10-01-preview' = [for item in keyData: {
+  name: '${configStoreName}/${(contains(item, 'label') ? '${item.key}$${item.label}' : item.key)}'
+  properties: {
+    value: item.value
+    contentType: (contains(item, 'contentType') ? item.contentType : json('null'))
+    tags: (contains(item, 'tag') ? item.tag : json('null'))
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep:53]: replace ```json('null')``` with ```null```

File: quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  sku: {
+    name: 'standard'
+  }
+}
+
+resource configStoreNameKeyVaule 'Microsoft.AppConfiguration/configurationStores/keyValues@2021-10-01-preview' = [for item in keyData: {
+  name: '${configStoreName}/${(contains(item, 'label') ? '${item.key}$${item.label}' : item.key)}'
+  properties: {
+    value: item.value
+    contentType: (contains(item, 'contentType') ? item.contentType : json('null'))
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep:52]: replace ```json('null')``` with ```null```

File: quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+resource configStore 'Microsoft.AppConfiguration/configurationStores@2021-10-01-preview' = {
+  name: configStoreName
+  location: location
+  sku: {
+    name: 'standard'
+  }
+}
+
+resource configStoreNameKeyVaule 'Microsoft.AppConfiguration/configurationStores/keyValues@2021-10-01-preview' = [for item in keyData: {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep:48]: use the parent syntax for the child resource

File: quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  name: '${configStoreName}/${(contains(item, 'label') ? '${item.key}$${item.label}' : item.key)}'
+  properties: {
+    value: item.value
+    contentType: (contains(item, 'contentType') ? item.contentType : json('null'))
+    tags: (contains(item, 'tag') ? item.tag : json('null'))
+  }
+  dependsOn: [
```
Comment (bmoore-msft) [inline quickstarts/microsoft.appconfiguration/app-configuration-store-kv-copy/main.bicep:55]: remove dependsOn when you've added the parent syntax

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**